src/app/experience/experience.component.html

```diff
@@ -1,9 +1,43 @@
+    </p>
+    <p class="experience-header">
+      June 2014 to Nov. 2015 - General Electric Capital: Java Software Engineer
+    </p>
+    <p>
+      Develop RESTful Java APIs for GE commercial real estate, develop continuous integration (CI) workflows, and implement robust data access layers. I have developed custom software to analyze log files from the production servers to better understand customer utilization in our Agile SDLC. Maintained legacy software application that was developed using multiple custom frameworks contributed new code using a standardized framework. This increased usability and maintainability of the software, while still delivering business value.
+    </p>
+    <p class="experience-header">
+      Aug. 2009 to May 2014 - The University of Texas Health Science Center at Houston: Graduate Student
+    </p>
+    <p>
+      From 2009 to 2014, I served as a researcher and system administrator for a small group (6 members) and maintained ~10 Linux desktop machines and a 396 CPU Linux cluster. This has given me exposure to several varieties of Linux (CentOS, RedHat & Ubuntu) and Windows Enterprise. I am also skilled at analyzing high dimensional data using unsupervised machine learning and chemistry based informatics. I am also a subject matter expert in computational chemistry and early stage computational drug discovery.
+    </p>
+    <p class="experience-header">
+      June 2007 to Aug. 2007 - Advanced Technology Development Facility: Lithography Summer Intern
+    </p>
+    <p>
+      During my 3-month internship, I was tasked with determining the best photoresist for patterning integrated circuits onto 200mm silicon wafers. I gained valuable experience into the early stages of integrated circuit production, and was trained to evaluate the quality of the imprinted circuit using high-resolution scanning electron microscopy (SEM).
+
+      I evaluated several photoresists for their ability to pattern integrated circuits (ICs) onto 200mm silicon wafers using 193-nm optical lithography. I gained valuable insight into the early stages of IC production, and was trained to evaluate the quality of the ICs using high-resolution scanning electron microscopy. Specifically, the photoresists were evaluated for their ability to generate dense and isolated features (iso/dense bias) as well as the linearity of the features. I was also trained to identify the ideal critical dimension (CD) and greatest depth-of-focus (DoF) by using focus exposure matrices (FEMs) and Bossung curves. Additionally, I learned about the theory of binary, attenuating and alternating phase shift masks.
+    </p>
+    <hr class="line">
     <div class="center-all-the-things">
-      <p>Here is a summary of my skill set with proficiencies from personal, educational and work experience:</p>
+      <p>Here is an overall summary of my skill set with proficiencies from personal, educational and work experience:</p>
       <table align="center">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "align" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3AS1827)